src/mixins/filing-template-mixin.ts

```diff
@@ -119,13 +118,13 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
       nameTranslations = this.prepareNameTranslations()
     }
 
-    // Build filing
+    // Build correction filing
     const filing: CorrectionFilingIF = {
       header: {
         name: FilingTypes.CORRECTION,
         certifiedBy: this.getCertifyState.certifiedBy,
-        date: this.getCurrentDate,
-        folioNumber: this.getFolioNumber
+        date: this.getCurrentDate, // "absolute day" (YYYY-MM-DD in Pacific time)
+        folioNumber: this.getStaffPayment.folioNumber // keep original folio number
```

Comment:
I'm going to revert this change ASAP. This should be the FN from the original IA unless staff override it on line 179 below.